src/SFML/Graphics/Font.cpp

```diff
@@ -83,13 +84,32 @@ namespace
 
 namespace sf
 {
+namespace priv
+{
+////////////////////////////////////////////////////////////
+struct FontHandles
+{
+private:
+    // Default constructible deleter functor
+    struct Deleter
+    {
+        void operator()(FT_Library library) { FT_Done_FreeType(library); }
+        void operator()(FT_Face face)       { FT_Done_Face(face); }
+        void operator()(FT_Stroker stroker) { FT_Stroker_Done(stroker); }
+    };
+
+public:
+    std::unique_ptr<std::remove_pointer_t<FT_Library>, Deleter> library   = nullptr; // Pointer to the internal library interface
+    std::unique_ptr<FT_StreamRec>                               streamRec = nullptr; // Pointer to the stream rec instance
+    std::unique_ptr<std::remove_pointer_t<FT_Face>,    Deleter> face      = nullptr; // Pointer to the internal font face
+    std::unique_ptr<std::remove_pointer_t<FT_Stroker>, Deleter> stroker   = nullptr; // Pointer to the stroker
```

Comment:
```suggestion
    std::unique_ptr<std::remove_pointer_t<FT_Library>, Deleter> library;   //< Pointer to the internal library interface
    std::unique_ptr<FT_StreamRec>                               streamRec; //< Pointer to the stream rec instance
    std::unique_ptr<std::remove_pointer_t<FT_Face>,    Deleter> face;      //< Pointer to the internal font face
    std::unique_ptr<std::remove_pointer_t<FT_Stroker>, Deleter> stroker;   //< Pointer to the stroker
```

Reply:
Is it guaranteed that `FT_XXX` types are always pointers on every platform? Could they be something else (e.g. handles)?

Reply:
It might be a hack to assume so because those types are documented as opaque handles, but looking at FreeType headers, they are indeed typedefs to pointer types regardless of the platform.

Reply:
I added some static assertions to make sure those types are pointer types.

Reply:
> I added some static assertions to make sure those types are pointer types.

That is not what I meant, I meant checking out the FreeType documentation. The code wouldn't compile with or without the static assertion. But if, for example `FT_Library` is a pointer on Win32 and a `int` on Linux, your approach wouldn't work.